config/config.go

```diff
@@ -25,21 +25,30 @@ import (
 	"fmt"
 	"log"
 	"os"
+	"time"
 
 	"gopkg.in/yaml.v3"
 )
 
 // a type with service configuration parameters
 type serviceConfig struct {
 	// port on which the service listens
-	Port int `json:"port" yaml:"port"`
+	Port int `json:"port,omitempty" yaml:"port,omitempty"`
```

Comment:
add in documentation for the default value?

Reply:
We could do this as part of #41.